mithril.js

```diff
@@ -1125,6 +1124,6 @@ m.redraw = redrawService.publish
 m.request = requestService.xhr
 m.jsonp = requestService.jsonp
 m.version = "1.0.0"
-module.exports = m
+window.m = m
```

Comment:
Oh, now that I think of it, having a `builder`-made CommonJS build could be of use if someone is looking for the absolute smallest file size post-browserify...